src/models/service.ts

```diff
@@ -97,7 +97,7 @@ const getServiceModel = ({
 		const { service } = await sdkInstance.models.application.get(
 			slugOrUuidOrId,
 			{
-				$select: 'service',
+				$select: 'id',
```

Comment:
The typings always attempted to block selecting reverse navigation resources but didn't work quite right, with the fixes it now blocks the select attempt as expected